.travis.yml

```diff
@@ -12,3 +12,5 @@ rvm:
   - jruby-19mode
   - jruby-9.1.12.0
 sudo: false
+before_script:
```

Comment:
The export does work (just confirmed with a test branch on my fork: https://travis-ci.org/pat/syntax/jobs/249047431), but will switch over to the other syntax anyway.